### ISSUE TITLE
Dependency listed in the wrong place

### ISSUE THREAD

**Issue by alerque** [wireservice/agate-excel#46]
https://github.com/wireservice/agate-excel/blob/5374a24ab2b2496a7acc98e666d1e2447fc5a41f/setup.py#L44

The user of `import six` is in the library itself, not in the test suite. This should be listed under `install_requires` not as a test requirement.

Also `nose` doesn't seem to be used at all.

**Comment by jpmckinney** [comment]
nose is used to run tests: https://github.com/wireservice/agate-excel/blob/7083d1e8be66ab01ae429f6f540f926b40b09317/.github/workflows/ci.yml#L29

I've moved `six` to `install_requires` and released a new version 0.2.5.

Thanks for the bug report!

**Comment by alerque** [comment]
Thanks for the fix on `six`. As for `nose`, I still assert that it is not used by either the library or the tests. It is used  by your CI to run your test suite, but it is not required by the tests themselves. I noticed this because I was able to run the test suite as part of the distro packaging check without it installed because by default we're using `pytest` to run the tests because it plays nice with the path requirements of the package build environment. In other words *some* test runner is required, but nose isn't specifically required. In any case since we don't read the test requirements anyway it is not a blocker, just worth a note.
